main.py

```diff
@@ -1,12 +1,243 @@
+    
+    return choice
+
+def run_cli(play=True):
+
+    #initialize video_store
+    video_store = VideoStore(url="https://retro-video-store-api.herokuapp.com")
+    
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, video_store)
+
+        video_store.print_selected()
+
+##### VIDEO CHOICES #####
+        if choice == '1':
+            print("Creating a new video record: \n")
+            title = input("What is the title of your video? ")
+            release_date = input("When was the release date of the video? ")
+            total_inventory = input("How many videos with this title are in stock? ")
+            response = video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_divider()
+            print("New video record created: \n", response["video"])
+
+        elif choice == '2':
+            print(f"Updating video record: {video_store.selected_video} \n")
+            title = input("What is the new title of video? ")
+            release_date = input("What is the new release date of the video? ")
+            response = video_store.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_divider()
+            print("Updated video:", response["video"])
+
+        elif choice == '3':
+            video_store.delete_video()
+
+            print_divider()
+            print("\nVideo record has been deleted.\n")
+
+            print_divider()
+            print(video_store.get_videos_list())
+
+        elif choice == '4':
+            print_divider()
+            print("Video Records: \n")
+            for video in video_store.get_videos_list():
+                print(video)
+
+        elif choice == '5':
+            select_by = input("Would you like to select by title or id: \n")
+            if select_by == "title":
+                title = input("Which video title would you like to select? ")
+                video_store.get_single_video(title=title)
+            elif select_by == "id":
+                id = input("Which video id would you like to select? ")
+                if id.isnumeric():
+                    id = int(id)
+                    video_store.get_single_video(id=id)
+            else:
+                print("Selection not valid. Please enter an id or title.\n")
+            
+            if video_store.selected_video:
+                print_divider()
+                print("Selected video: ", video_store.selected_video)
+
+##### CUSTOMER CHOICES #####
+        elif choice == '6':
+            print("Creating a new customer record: \n")
+            name = input("What is the customer's name? ")
+            postal_code = input("What is the customer's postal code? ")
+            phone = input("What is the customer's phone number? ")
+            registered_at = input("When was the customer registered? ")
```

Comment:
The value registered_at is created by the API when the customer is added to the system, and does not need to be inputed by the user.